Source/Documentation/Manual/physics.rst

```diff
@@ -2246,6 +2246,9 @@ The brake can be applied by pressing ], and released by pressing the [ keys.
 To control the application and release rates on the brake use the ``EngineBrakesControllerMaxApplicationRate`` and 
 ``EngineBrakesControllerMaxReleaseRate`` parameters.
 
+A steam effect can be added by adding the following special effects to the wagon ``SteamBrakeFX``. This special effect will 
+turn on and off with the brake operation and can be used to model steam leakage of the steam brake cylinder, etc.
```

Comment:
```suggestion
The ``SteamBrakeFX`` special effect, if added to the wagon, will turn on and 
off with the brake operation and can be used to model steam leakage of the 
steam brake cylinder, etc.
```
I suggest rewording this to flow more naturally.

Reply:
Suggested change added.